Poker.BE/Poker.BE.Domain/Game/Player.cs

```diff
@@ -13,24 +14,40 @@ public enum State
         {
             ActiveUnfolded,
             ActiveFolded,
+            ActiveAllIn,
             Passive
         }
 
         public static readonly int NPRIVATE_CARDS = 2;
         #endregion
 
         #region Properties
-        public State CurrentState { get; private set; }
+        public State CurrentState { get; set; }
```

Comment:
We need to let the round (or turn) to be able to change the player's state according to his game play: if he folds or goes all-in.
All-in is actually not a problem because you can change the state from inside the player when all of his money is in the pot, but the "fold" play move doesn't have a special sign (like all-in's money=0) so that's more of a problem...